src/exports.js

```diff
@@ -3,10 +3,12 @@ import ts from "./components/tab-switcher/tab-switcher.jsx";
 import t from "./components/tab-switcher/tab.jsx";
 import rf from "./components/radio-filter/radio-filter.jsx";
 import c from "rc-collapse";
+import mf from "./components/mofo-footer/mofo-footer.jsx";
```

Comment:
Do we need to have a `mofo-footer` dir here? There's only 1 file (`mofo-footer.jsx`) in that dir.